Assets/PatchKit Patcher/Scripts/PatchKitLogo.cs

```diff
@@ -0,0 +1,82 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using System.Collections;
+using UniRx;
+using PatchKit.Unity.Patcher.Debug;
+
+namespace PatchKit.Unity
+{
+
+    public class PatchKitLogo : MonoBehaviour
+    {
+        public string patchKitWebsiteUrl;
+
+        public Texture2D cursorTexture;
+
+        public Vector2 cursorHotspot;
+
+        public Image image;
+
+        public Button button;
+
+        private bool _isLogoVisible = false;
+
+        private const float LogoVisibilityChangeDelay = 3; // in seconds
+
+        private void Awake()
+        {
+            Assert.IsNotNull(button);
+            Assert.IsNotNull(image);
+
+            Assert.IsNotNull(cursorTexture);
+
+            button.enabled = false;
```

Comment:
Avoid using other components/objects inside `Awake`, use `Start` instead.

(for example if someone would like to manually add this component, he wouldn't have a chance to set `Button` reference)